### PR TITLE
Add pricing and more overview information

### PR DESCRIPTION
**Pull request by choldgraf**
This adds some more core information about the Hub Service to our documentation. It adds these major sections:

- Pricing guidelines
  - This includes a very rough cloud costs calculator. It tries to overestimate the costs a bit.
- An overview of what "DevOps as a Service" things we provide
- An overview of roles that are needed in a hub
- A page with terminology and background

### CODE REVIEW

**Comment by choldgraf** [comment]
I would love thoughts from @yuvipanda about whether the cloud costs calculator is a reasonable over-estimate of the costs for each hub. I tried to make it clear that those estimates are **very** rough, but the main point is to help people understand what order of magnitude they should expect in cloud costs.

I'd also love thoughts from @colliand about whether this further clarifies our service offerings, and gives "decision makers" in an organization more ammo they can use to advocate for our service.

File: about/pricing.md
Comment (damianavila) [inline about/pricing.md:58]: OK, all this section is heavily biased towards the GCP stuff... I understand we should start with something but can we add, at least, a warning about estimates differences by cloud providers?

Comment (choldgraf) [reply about/pricing.md:58]: Good point - i was just using it as a single estimate to avoid overloading people with information. I'll add a note about comparable machines for the other providers (and the z2jh link has a lot of good resources too)

Comment (choldgraf) [reply about/pricing.md:69]: I've added a note about this at the beginning of the section

**Review by damianavila — APPROVED** [review]
LGTM, but before merging it, I would wait for feedback from another team member more familiarized with the GCP cost modeling.

**Comment by choldgraf** [comment]
good point - any chance that @yuvipanda would be willing to give a quick look-over this section:

https://github.com/2i2c-org/pilot/pull/99/files#diff-ca7035b71ea190fbedfcd6c2e086bed02a251c2f37a7051483caff053376714fR39

and make sure that I am not making a big mistake there?

**Comment by choldgraf** [comment]
Hey all - I haven't gotten any comments on the GCP pricing stuff yet. If nobody gives me a 👍 on that, then today I am going to remove that section from these docs and then merge this PR. My bad for trying to do too many things in one PR.

**Comment by choldgraf** [comment]
I've commented out all of the specific numbers calculation for GCP, and instead have added in a few links to pre-existing blog posts and analyses. That way we aren't making statements that are quite as strong, and I think it's easier for users to think of them as estimates rather than guarantees.

I'll merge this in, and we can uncomment the example section once we're more comfortable with our numbers!

**Pull request merged**
